src/utils.js

```diff
@@ -282,9 +282,21 @@ const recapAlertButton = (court, pacerCaseId, isActive) => {
   url.searchParams.append('pacer_case_id', pacerCaseId);
   url.searchParams.append('court_id', court);
   anchor.href = url.toString();
+  
   const img = document.createElement('img');
   img.src = chrome.extension.getURL(`assets/images/${icon}-16.png`);
-  anchor.innerHTML = `${img.outerHTML} ${text}`;
+
+  const p = document.createElement('p')
+  p.innerHTML = text
+  
+  const innerDiv = document.createElement('div')
+  innerDiv.classList.add("recap-banner-inner-div");
+  
+  innerDiv.appendChild(img)
+  innerDiv.appendChild(p)
+
+  anchor.append(innerDiv)
+  
```

Comment:
Feels like this (and possibly more?) could be in a little utility function.

Reply:
You're right! I've moved this code to an utility function.